src/main/java/core/basesyntax/SimpleFileEditor.java

```diff
@@ -0,0 +1,242 @@
+        switch (command) {
+            case CREATE:
+                System.out.println("create : to create file, need to specify [path] [file-name]!");
+                break;
+            case READ:
+                System.out.println("read : to read file, need to specify [path] [file-name]!");
+                break;
+            case INFO:
+                System.out.println("info : Displays brief information on the specified file:"
+                        + "number of characters, lines, words, date and time of"
+                        + "the last change, file size");
+                break;
+            case EXIT:
+                System.out.println("exit : Exit from app!");
+                break;
+            default:
+                System.out.println("Unknown command!");
+                break;
+        }
+    }
+
+    private void infoAboutFile(String[] wordsCommand) {
+        if (validate(wordsCommand)) {
+            String pathString = wordsCommand[1].substring(1, wordsCommand[1].length() - 1);
+            String fileNameString = wordsCommand[2].substring(1, wordsCommand[2].length() - 1);
+            Path path = Paths.get(pathString + "\\" + fileNameString);
+            StringBuilder text = new StringBuilder();
+            int counterLine = 0;
+            try (BufferedReader bufferedReader = Files.newBufferedReader(path)) {
+                String line;
+                while ((line = bufferedReader.readLine()) != null) {
+                    text.append(line).append(' ');
+                    counterLine++;
+                }
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+            String[] words = text.toString().split(" ");
+            System.out.println("Quantity characters : " + text.length());
+            System.out.println("Quantity words : " + words.length);
+            System.out.println("Quantity lines : " + counterLine);
+            try {
+                System.out.println("Last modified : " + Files.getLastModifiedTime(path));
+                System.out.println(String.format("File size : %d bite.", Files.size(path)));
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    private void createFile(String[] wordsCommand) {
+        if (validate(wordsCommand)) {
+            String pathString = wordsCommand[1].substring(1, wordsCommand[1].length() - 1);
+            String fileNameString = wordsCommand[2].substring(1, wordsCommand[2].length() - 1);
+            Path path = Paths.get(pathString + "\\" + fileNameString);
+            if (!Files.exists(Paths.get(pathString))) {
+                System.out.println("Path not exists!");
+                return;
+            }
+            if (Files.exists(path)) {
+                System.out.println("File exists! Overwrite ?");
+                String answer = "";
+                while (!answer.equals(YES)) {
```

Comment:
reconsider this condition. First you check that answer should not be equal to Yes and then if it's equal. I understood how it works in your case, but that took some time. Try to refactor this part and make it more understandable.